### ISSUE TITLE
Pydantic v2.0.0 import error

### ISSUE THREAD

**Issue by kevinschaich** [RobertCraigie/prisma-client-py#778]
## Bug description

Just got this error upon installation a few minutes ago:

```
Traceback (most recent call last):
  File "/usr/local/bin/prisma-client-py", line 5, in <module>
    from prisma.cli import main
  File "/usr/local/lib/python3.11/dist-packages/prisma/__init__.py", line 11, in <module>
    from ._config import config as config
  File "/usr/local/lib/python3.11/dist-packages/prisma/_config.py", line 7, in <module>
    from pydantic import BaseSettings, Extra, Field
  File "/usr/local/lib/python3.11/dist-packages/pydantic/__init__.py", line 206, in __getattr__
    return _getattr_migration(attr_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/pydantic/_migration.py", line 279, in wrapper
    raise PydanticImportError(
pydantic.errors.PydanticImportError: `BaseSettings` has been moved to the `pydantic-settings` package. See https://docs.pydantic.dev/2.0/migration/#basesettings-has-moved-to-pydantic-settings for more details.
```

I think version 2.0.0 got released like 30 minutes ago 😅 

## Expected behavior

I think we should pin a version dependency on either a `1.x.x` version in `prisma-client-py` or update the imports as described in that link above.

## Environment & setup

- OS: MacOS
- Database: Postgres
- Python version: 3.11
- Prisma version: 4.15.0

```
prisma py version
prisma                  : 4.15.0
prisma client python    : 0.9.0
platform                : darwin
expected engine version : 8fbc245156db7124f997f4cecdd8d1219e360944
installed extras        : []
install path            : /opt/homebrew/lib/python3.11/site-packages/prisma
binary cache dir        : /Users/kevinschaich/.cache/prisma-python/binaries/4.15.0/8fbc245156db7124f997f4cecdd8d1219e360944
```

**Comment by blinton-sc** [comment]
Yeah this broke it for us as well.

Should change this line: https://github.com/RobertCraigie/prisma-client-py/blob/main/requirements/base.txt#L3 to `pydantic==1.10.6`

**Comment by tooruu** [comment]
You shouldn't even be using `>=` in the first place when `~=` exists.
```suggestion
pydantic~=1.10.0
```

**Comment by RobertCraigie** [comment]
> You shouldn't even be using >= in the first place when ~= exists.

No, this is a common bad practice. Premature dependency pinning can cause even more annoying issues, see this blog post for reasoning: https://hynek.me/articles/semver-will-not-save-you/

Especially this part:

> On the other hand, if a new major version of a package surprisingly breaks your package, they can always add a pin themselves (see step 4 above) until you fix your package. But there’s no practical way for them to remove your pin.

If you want to avoid this kind of problem in the future I recommend https://github.com/jazzband/pip-tools, otherwise this issue should only affect new projects.

---

Sorry for the troubles caused here! I'll be pushing a new release shortly.

**Comment by RobertCraigie** [comment]
I've published `v0.9.1` which pins `pydantic`.
